examples/http-caching-client/http_caching_client.bal

```diff
@@ -10,15 +10,14 @@ import ballerina/http;
 // the `cacheConfig`. Currently, there are only 2 policies:
 // `CACHE_CONTROL_AND_VALIDATORS` (the default policy) and `RFC_7234`.
 
-http:Client cachingEP = checkpanic new ("http://localhost:8080",
-                        {cache: {enabled: true, isShared: true}});
+http:Client cachingEP = checkpanic new ("http://localhost:8080", 
+    cache = {enabled: true, isShared: true});
 service / on new http:Listener(9090) {
 
     resource function get cache(http:Request req)
             returns http:Response|error? {
```

Comment:
Move this to above line?

Reply:
I think it violates the character limit though. Will check again.